subsys/bluetooth/mesh/access.c

```diff
@@ -1145,7 +1145,7 @@ static void store_pending_mod_sub(struct bt_mesh_model *mod, bool vnd)
 
 static void store_pending_mod_pub(struct bt_mesh_model *mod, bool vnd)
 {
-	struct mod_pub_val pub;
+	struct mod_pub_val pub = {0};
```

Comment:
`pub.uuid = NULL` after line 1163?

Reply:
`struct mod_pub_val` doesn't have `uuid`. It is related to bitfileds `period_div` and `cred` and unused part of them

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/54823#discussion_r1106020761